### PR TITLE
Feature/search

### PR DESCRIPTION
**Pull request by ybutrameev**

### CODE REVIEW

File: src/app/component/SearchBar/SearchBar.component.js [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+            <p block="SearchBar" elem="NoResults">No products found!</p>
+        );
+    }
+
+    renderResults(items) {
+        return (
+            <ul>
+                {
+                    items && items.map((product, key) => (
+                        key < 20 && (
```
Comment (alfredsgenkins) [inline src/app/component/SearchBar/SearchBar.component.js]: What is this, any comments, please?

File: src/app/component/SearchBar/SearchBar.style.scss [hunk 2]
```diff
@@ -0,0 +1,281 @@
+.SearchBar {
+    --border-color: var(--header-color);
```
Comment (alfredsgenkins) [inline src/app/component/SearchBar/SearchBar.style.scss]: Please make it more specific. Add it to root.scss

File: src/app/component/SearchBar/SearchBar.style.scss [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+            width: 25px;
+            height: 4px;
+            display: block;
+            float: right;
+            transition: all 350ms cubic-bezier(0.25, 1.7, 0.35, 0.8);
+            z-index: -1;            
+        }
+
+        &_visible {
+            &:after {
```
Comment (alfredsgenkins) [inline src/app/component/SearchBar/SearchBar.style.scss]: Always use `::after` for pseudo elements!!!

File: src/app/component/Header/Header.style.scss [hunk 4]
```diff
@@ -34,11 +34,11 @@
         padding: 1rem;
         display: grid;
         margin:auto;
-        grid-template-columns: 1fr 1.3fr 1fr;
+        grid-template-columns: 1fr 40rem 1fr;
         max-width: 1200px;
 
         @include mobile {
-            grid-template-columns: 1.1fr 0.7fr 2.7fr 1.8fr;
+            grid-template-columns: 4rem 4rem 9rem 8rem;
```
Comment (alfredsgenkins) [inline src/app/component/Header/Header.style.scss]: Are you sure you want ALL columns to be fixed width?

File: src/app/component/SearchBar/SearchBar.style.scss [hunk 5]
```diff
@@ -1,14 +1,23 @@
...
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
 .SearchBar {
-    --border-color: var(--header-color);
```
Comment (alfredsgenkins) [inline src/app/component/SearchBar/SearchBar.style.scss]: What color is this?

**Comment by alfredsgenkins** [comment]
Please create an issue and mention it by writing `fixes #[issue number]`.

**Comment by ybutrameev** [comment]
fixes #35

**Comment by ybutrameev** [comment]
Don't merge this!

**Comment by ybutrameev** [comment]
> Don't merge this!

Now it is ready

**Comment by alfredsgenkins** [comment]
Fixes #35

**Comment by alfredsgenkins** [comment]
@all-contributors please add @ybutrameev for code, bug

**Pull request merged**